test/test.dart

```diff
@@ -383,8 +377,7 @@ Future<Realm> getIntegrationRealm({App? app, ObjectId? differentiator, String? p
   app ??= App(await getAppConfig());
   final user = await getIntegrationUser(app);
 
-  // TODO: path will not be needed after https://github.com/realm/realm-dart/pull/574
-  final config = Configuration.flexibleSync(user, [Task.schema, Schedule.schema, NullableTypes.schema], path: path);
+  final config = Configuration.flexibleSync(user, [Task.schema, Schedule.schema, NullableTypes.schema]);
```

Comment:
Are we sure we don't want to allow users to set the path explicitly? One thing is to have a sane default, another is to prevent the user from choosing himself?

Reply:
This was a stale TODO that I fixed. since #574 was merged. So I guess either I should be removing the path argument or keeping it?

Reply:
Sorry.. I read this wrong. For a minute there I thought you removed the argument on the ctor 🙄

Reply:
no it was good. I removed this path argument on multiple places which were leftovers